src/Xamarin.Android.Build.Tasks/Utilities/FileResourceParser.cs

```diff
@@ -71,6 +73,38 @@ public IList<R> Parse (string resourceDirectory, IEnumerable<string> additionalR
 					Log.LogDebugMessage ($"Skipping non-existent directory: {dir}");
 				}
 			}
+			foreach (var aar in aarLibraries ??  Array.Empty<string>()) {
+				Log.LogDebugMessage ($"Processing Aar file {aar}");
+				if (!File.Exists (aar)) {
+					Log.LogDebugMessage ($"Skipping non-existent aar: {aar}");
+					continue;
+				}
+				using (var file = File.Open (aar, FileMode.Open, FileAccess.ReadWrite, FileShare.Read)) {
+					using var zip = ZipArchive.Open (file);
```

Comment:
Given #9623 and the apparent *immense* performance difference between Xamarin.Tools.Zip and System.IO.Compression, I think we should try using System.IO.Compression to read the `.aar` files, unless we require Xamarin.Tools.Zip for some reason…